src/app.ts

```diff
@@ -27,9 +34,33 @@ function startApiServer() {
 
   app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerConfig));
 
-  app.listen(port);
+  server = app.listen(port);
+
+  await axios.post('http://82.140.0.78:9092/kapacitor/v1/alerts/topics/cpu/handlers', {
+    id: handlerId,
+    kind: 'post',
+    options: {
+      url: `http://${getVpnAddress()}:${port}/`,
+    },
+  });
 }
 
+process.on('SIGINT', async () => {
+  console.log('Deleting kapacitor handler.');
```

Comment:
TSLint outputs: `WARNING: ../monitoring-backend/src/app.ts:49:3 - Calls to 'console.log' are not allowed.`

Can we use winston here?